docs/ref/databases.txt

```diff
@@ -244,6 +244,30 @@ database configuration in :setting:`DATABASES`::
         },
     }
 
+.. _postgresql-pool:
+
+Connection Pool
+---------------
+
+To use a connection pool with `psycopg`_ 3.1.8+, you can either set ``"pool"``
+in the :setting:`OPTIONS` part of your database configuration in
+:setting:`DATABASES` to be ``True`` or as a dict to be passed to
```

Comment:
Maybe worth mentioning that `True` means the psycopg defaults are used?

Reply:
Good suggestion thank you, added in ⭐